src/ml_tooling/baseclass.py

```diff
@@ -65,73 +72,89 @@ def estimator_name(self):
         return class_name
 
     @classmethod
-    def load_estimator(cls, path: str) -> "Model":
+    def list_estimators(cls, storage: Storage) -> List[pathlib.Path]:
```

Comment:
Why make it a class method when it doesn't creates an instance of the Model class?

Reply:
Should maybe be a static function. It doesn't make much sense to first initiated it with one estimator and then get a list of other estimators.

Reply:
yes, this should be static indeed :)